test/unit/krane/ejson_secret_provisioner_test.rb

```diff
@@ -133,7 +133,7 @@ def test_run_with_selector_does_not_raise_exception
   private
 
   def stub_server_dry_run_validation_request(attempts: 3)
-    stub_kubectl_response("apply", "-f", anything, "--server-dry-run", "--output=name",
+    stub_kubectl_response("apply", "-f", anything, "--dry-run=server", "--output=name",
```

Comment:
This is because I changed the dummy data below to use a version > 1.18. Krane doesn't support 1.18 anymore, so the handling of the older flag could be removed entirely at this point (unrelated to this PR).